packages/mongodb-memory-server-core/src/MongoMemoryReplSet.ts

```diff
@@ -253,13 +253,7 @@ export default class MongoMemoryReplSet extends EventEmitter {
 
       // MongoClient HACK which helps to avoid the following error:
```

Comment:
i still let a bit of the old hack-comment to know what this line was for and for what issues to look out for